dataprofiler/tests/profilers/test_graph_profiler.py

```diff
@@ -0,0 +1,146 @@
+from __future__ import print_function
+
+import os
+import unittest
+from cgi import test
+
+import networkx as nx
+
+from dataprofiler.profilers.graph_profiler import GraphProfile
+
+test_root_path = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
+
+
+class TestGraphProfiler(unittest.TestCase):
+    def setUp(self):
+        self.graph = nx.Graph()
```

Comment:
we need to ensure the GraphProfile can ingest a `GraphData`, hence ideal if we setup our own GraphData class.
We could mock it such that create a GraphData Class and set the values to be this graph.

Reply:
or mock the class